src/platforms/qurt/px4_layer/commands_hil.c

```diff
@@ -49,6 +49,7 @@ const char *get_commands()
 		"param set CAL_MAG0_ID 196608\n"
 //		"rgbled start\n"
 //		"tone_alarm start\n"
+		"load_mon start\n"
```

Comment:
You also need to add to the startup file for on the Snapdragon DSP in https://github.com/PX4/Firmware/blob/master/posix-configs/eagle/flight/px4-flight.config.